Lib/idlelib/configdialog.py

```diff
@@ -570,9 +571,14 @@ def create_page_font_tab(self):
         indent_title = Label(
                 frame_indent, justify=LEFT,
                 text='Python Standard: 4 Spaces!')
-        self.indent_scale = Scale(
-                frame_indent, variable=self.space_num,
-                orient='horizontal', tickinterval=2, from_=2, to=16)
+        try:
+            self.indent_chooser = Spinbox(
+                    frame_indent, textvariable=self.space_num,
+                    state="readonly", from_=2, to=16)
+        except TclError:
+            self.indent_chooser = Combobox(
+                    frame_indent, textvariable=self.space_num,
+                    state="readonly", values=list(range(2,17)))
```

Comment:
I would like to delete this, at least on 3.10+, by requiring 8.5.9+  (even that is hardly usable on Apple).  Will probably ask on pydev.